src/toil/cwl/cwltoil.py

```diff
@@ -1070,6 +1076,14 @@ def main(args=None, stdout=sys.stdout):
     parser.add_argument(
         "--no-read-only", action="store_true", default=False,
         help="Do not set root directory in the container as read-only")
+    parser.add_argument(
+        "--cachedir", type=Text, default="",
```

Comment:
Did you test this?

Reply:
This needs testing, I doubt it will work

Reply:
I would remove this option, as it doesn't work. It would need to be refactored to support the toil file store.

```
$ toil-cwl-runner --cachedir=$PWD/cache  v1.0/v1.0/binding-test.cwl v1.0/v1.0/bwa-mem-job.json
[…]
WARNING:toil.leader:kind-file_home_michael_common-workflow-language_v1.0_v1.0_binding-test.cwl/instance7ll0hobk    INFO:toil.worker:---TOIL WORKER OUTPUT LOG---
WARNING:toil.leader:kind-file_home_michael_common-workflow-language_v1.0_v1.0_binding-test.cwl/instance7ll0hobk    INFO:toil:Running Toil version 3.21.0a1-2a50150c036d0bff86125c7ab4ba71c78524c836.
WARNING:toil.leader:kind-file_home_michael_common-workflow-language_v1.0_v1.0_binding-test.cwl/instance7ll0hobk    WARNING:toil.fileStore:Starting job files/for-job/kind-file_home_michael_common-workflow-language_v1.0_v1.0_binding-test.cwl/instance7ll0hobk/cleanup/file-e312hfda/stream with less than 10% of disk space remaining.
WARNING:toil.leader:kind-file_home_michael_common-workflow-language_v1.0_v1.0_binding-test.cwl/instance7ll0hobk    Traceback (most recent call last):
WARNING:toil.leader:kind-file_home_michael_common-workflow-language_v1.0_v1.0_binding-test.cwl/instance7ll0hobk      File "/home/michael/src/toil/env3.7/lib/python3.7/site-packages/cwltool/pathmapper.py", line 297, in visit
WARNING:toil.leader:kind-file_home_michael_common-workflow-language_v1.0_v1.0_binding-test.cwl/instance7ll0hobk        st = os.lstat(deref)
WARNING:toil.leader:kind-file_home_michael_common-workflow-language_v1.0_v1.0_binding-test.cwl/instance7ll0hobk    FileNotFoundError: [Errno 2] No such file or directory: '/tmp/toil-35b2d62d-75c5-44d7-8232-483b20b6e316-8f41ddb6e30e4bc0b9adde0837fcd6eb/tmpx2_b3xtp/b63c25f9-4058-42ec-9e42-edd6ea230b48/toilfs:files/no-job/file-tz1oe3fn/chr20.fa'
```

Reply:
My bad! Updated this to remove it in the recent commit.